src/software/ai/navigator/obstacle/robot_navigation_obstacle_factory.cpp

```diff
@@ -182,6 +182,13 @@ ObstaclePtr RobotNavigationObstacleFactory::createFromShape(const Circle &circle
 
 ObstaclePtr RobotNavigationObstacleFactory::createFromShape(const Polygon &polygon) const
 {
+    // modified for debugging
```

Comment:
just wanted to check, did you want this debugging statement to continue being in the code? If so, should you use the logger and replace the return statement to use "poly"

Reply:
Thanks for pointing it out. I've removed it as it was unnecessary.